### PR TITLE
Update with latest deploy details

### PR DESCRIPTION
**Pull request by chris48s**
Couldn't push your "Lots of changes" commit straight onto master as I don't have commit access to `DemocracyClub/polling_deploy` so I've bundled it in here and made my changes on top of it. The amount of stuff I actually changed yesterday is fairly minimal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,17 @@ amis.
 
 - If the addressbase dump has changed, or if you haven't built it yet then:
 
+#### 1. Create a DB dump containing addressbase
```
Comment (symroe) [inline README.md]: This is no longer true now we process AddressBase on the server

File: README.md [hunk 2]
```diff
@@ -18,6 +18,17 @@ amis.
...
+
+You will need a local working copy of the app code and a fresh copy of addressbase (basic) in CSV files. clean and import address base according to the app's documentation, at the moment this involves running `./manage.py clean_addressbase` and `./manage.py import_cleaned_addresses`.
+
+#### 2. Export and upload addressbase
```
Comment (symroe) [inline README.md]: This is no longer true now we process AddressBase on the server

File: packer.json [hunk 3]
```diff
@@ -4,12 +4,16 @@
     "build_region": "eu-west-1",
     "regions_to_copy_to": "eu-central-1",
     "branch": "master",
-    "max_spot_price": "0.1"
+    "max_spot_price": "0.1",
```
Comment (symroe) [inline packer.json]: Might be worth making this `0.2` as we had to wait a while last time for a `0.1` to show up

**Comment by chris48s** [comment]
That should all be right now

**Pull request merged**